<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackControlWindow</class>
 <widget class="QWidget" name="TrackControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>d-jon fast control</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../djon_rs.qrc">
    <normaloff>:/img/djon.png</normaloff>:/img/djon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="actionButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../djon_rs.qrc">
          <normaloff>:/img/start.png</normaloff>:/img/start.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEdit">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string>h:mm:ss</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TreeComboBox" name="comboBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeComboBox</class>
   <extends>QComboBox</extends>
   <header>treecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../djon_rs.qrc"/>
 </resources>
 <connections/>
</ui>
